fx_crash_sig/cmd_get_crash_sig.py

```diff
@@ -33,12 +33,17 @@ def cmdline():
 
     crash_processor = CrashProcessor(verbose=args.verbose,
                                      windows=args.windows)
+
+    if sys.stdin.isatty():
+        print('fx-crash-sig: Failed: pass crash trace using stdin')
+        sys.exit(1)
+
```

Comment:
This checks to see if something is being piped in via stdin and if not, prints and error and exits. This prevents the case where someone runs `fx-crash-sig` and it just hangs which isn't very informative.

If I had my druthers, I'd redo this command to use click and make it more user-friendly. But I'm not sure how much value we get out of that since it's not clear who's using this as installed from PyPI.

Reply:
Makes sense. Personally I think we should just stop uploading to pypi until we have a clear use case for it.

Reply:
:+1: Less is more.